client/internal/peer/worker_relay.go

```diff
@@ -74,7 +74,7 @@ func (w *WorkerRelay) OnNewOffer(remoteOfferAnswer *OfferAnswer) {
 	relayedConn, err := w.relayManager.OpenConn(srv, w.config.Key)
 	if err != nil {
 		if errors.Is(err, relayClient.ErrConnAlreadyExists) {
-			w.log.Infof("do not need to reopen relay connection")
+			w.log.Debugf("reusing relay connection")
```

Comment:
This msg is not fully correct because we ignored this offer from a relay perspective, not reusing the existing connection. So we will get an Offer because of ICE and if this offer is irrelevant for Relay (because we already opened the connection) then we will do nothing.